ios/storage.md

```diff
@@ -101,37 +101,39 @@ open <YOURAPP>.xcworkspace
 Initialize `AWSMobileClient`, `Amplify`, and `AWSS3StoragePlugin`.
 
 Add the following imports to the top of your `AppDelegate.swift` file
-    ```swift
-    import Amplify
-    import AWSMobileClient
-    import AmplifyPlugins
-    ```
-Add the following code to your AppDelegate's `application:didFinishLaunchingWithOptions` method
 
-    ```swift
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
+```swift
+import Amplify
+import AWSMobileClient
+import AmplifyPlugins
+```
+    
+Add the following code to your AppDelegate's `application:didFinishLaunchingWithOptions` method
 
-        AWSMobileClient.default().initialize { (userState, error) in
-            guard error == nil else {
-                print("Error initializing AWSMobileClient. Error: \(error!.localizedDescription)")
-                return
-            }
-            print("AWSMobileClient initialized, userstate: \(userState)")
-        }
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    // Override point for customization after application launch.
 
-        let storagePlugin = AWSS3StoragePlugin()
-        do {
-            try Amplify.add(plugin: storagePlugin)
-            try Amplify.configure()
-            print("Amplify configured with storage plugin")
-        } catch {
-            print("Failed to initialize Amplify with \(error)")
+    AWSMobileClient.default().initialize { (userState, error) in
+        guard error == nil else {
+            print("Error initializing AWSMobileClient. Error: \(error!.localizedDescription)")
+            return
         }
+        print("AWSMobileClient initialized, userstate: \(userState)")
+    }
```

Comment:
Seems like we should wait for initialize to properly come up prior to creating the storage plugin etc.  So maybe the code should be something like:
```
    func application(_ application:..........) -> Bool {
        AWSMobileClient.default().initialize { (userState, error) in
                guard error == nil else {
                   .........
                 }
                 self.startAmplify()
        }
    }
    func startAmplify() {
            let storagePlugin = AWSS3StoragePlugin()
             do {
                   try Amplify.add(.......
                ......
             }
    }
```

Reply:
good point, updating the code